pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright © 2020 Dell Inc. or its subsidiaries. All Rights Reserved.
+Copyright © 2020-2024 Dell Inc. or its subsidiaries. All Rights Reserved.
 
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
```

Comment:
to fix the UT error, we may remove the tested case for this part of added back code which was removed in previous PR. 
https://github.com/dell/csi-baremetal/blob/rel/CY2310/pkg/base/linuxutils/lsblk/lsblk_test.go#L88C1-L94C28

Reply:
yep removed it